cypress/page/products-list.page.ts

```diff
@@ -2,7 +2,8 @@ class ProductsListPage {
     private addToCartBtn: string;
 
     constructor() {
-        this.addToCartBtn = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
+        // Eliminate .btn.btn-default, this is innecesary because with the previous selectors we already reach the desired button
+        this.addToCartBtn = "#center_column a.button.ajax_add_to_cart_button";
```

Comment:
I think you don't need the center column id, as the class is already explicit enough
```suggestion
        this.addToCartBtn = ".ajax_add_to_cart_button";
```